Clinical-Genomics/cg
# Add support for new microbial OF 1603:8

This PR Add support for new microbial OF 1603:8

How to test:
0. install clinical-api on clinical-db stage
1. login to OP
1. create order with name #123456
1. use the OF 1603:8 to submit some samples in OP

Expected outcome:
The samples end up in the status-db 

- [x] Deploy by @ingkebil

This a minor update since it only a change to which version of the OF we accept
